Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM python:3.9-slim
+FROM python:3.12-slim-bookworm
 
-LABEL vendor=neon.ai \
-    ai.neon.name="pyklatchat-client"
+ENV OVOS_CONFIG_BASE_FOLDER=neon
+ENV OVOS_CONFIG_FILENAME=klat.yaml
+ENV XDG_CONFIG_HOME=/config
+ENV KLAT_ENV=PROD
 
-ENV OVOS_CONFIG_BASE_FOLDER neon
-ENV OVOS_CONFIG_FILENAME diana.yaml
-ENV XDG_CONFIG_HOME /config
-COPY docker_overlay/ /
```

Comment:
This should probably be replaced so default configuration is available. Looks like that config file also needs to be refactored from `diana.yaml` to `klat.yaml`